app/templates/components/paper-menu-content.hbs

```diff
@@ -5,8 +5,8 @@
       class="ember-basic-dropdown-content {{class}} {{if renderInPlace 'ember-basic-dropdown-content--in-place'}} {{if hPosition (concat 'ember-basic-dropdown-content--' hPosition)}} {{if vPosition (concat 'ember-basic-dropdown-content--' vPosition)}}
       md-open-menu-container md-whiteframe-z2 md-clickable {{if isActive "md-active"}}"
       dir={{dir}}>
-      {{#paper-menu-content-inner width=width dense=dense dropdown=dropdown}}
-        {{yield}}
+      {{#paper-menu-content-inner width=width dense=dense dropdown=dropdown as |content|}}
+        {{yield (hash menu-item=(component 'paper-menu-item' parentComponent=content))}}
```

Comment:
Here we would just "re-yield" using `{{yield content.paper-menu-item}}` or something similar.